src/main/java/org/ibp/api/java/impl/middleware/inventory/manager/LotTemplateExportServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package org.ibp.api.java.impl.middleware.inventory.manager;
+
+import com.google.common.io.Files;
+import org.ibp.api.domain.location.LocationDto;
+import org.ibp.api.domain.ontology.VariableDetails;
+import org.ibp.api.exception.ResourceNotFoundException;
+import org.ibp.api.java.inventory.manager.LotTemplateExportService;
+import org.springframework.stereotype.Service;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.MapBindingResult;
+
+import javax.annotation.Resource;
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+
+@Service
+public class LotTemplateExportServiceImpl implements LotTemplateExportService {
```

Comment:
@cuenyad You have also renamed the class, this class must refer to Excel, please revert the change in the class name